### PR TITLE
set device name to Chrome

### PR DESCRIPTION
**Pull request by jonahwilliams**
## Description
More accurately reflect the status of the "web" devices as a chrome instance.

### CODE REVIEW

**Review by goderbauer — APPROVED** [review]
LGTM

No tests are failing because of this?

**Review by zanderso — APPROVED** [review]

**Comment by jonahwilliams** [comment]
Smoke test added for name and other fields

**Comment by DanTup** [comment]
In the future would we have other browsers appear as devices? If so, should the ID by something more-specific to Chrome too? (we do have categories on there to know it's a web device).

File: packages/flutter_tools/lib/src/web/web_device.dart [hunk 1]
```diff
@@ -74,7 +74,7 @@ class WebDevice extends Device {
   bool isSupported() => flutterWebEnabled && canFindChrome();
 
   @override
-  String get name => 'web';
+  String get name => 'Chrome';
```
Comment (devoncarew) [inline packages/flutter_tools/lib/src/web/web_device.dart]: 👍 thanks!

File: packages/flutter_tools/test/web/devices_test.dart [hunk 2]
```diff
@@ -26,6 +26,20 @@ void main() {
         return null;
       });
     });
+
+    test('Defaults', () async {
+      final WebDevice device = WebDevice();
+
+      expect(device.name, 'Chrome');
+      expect(device.id, 'web');
```
Comment (devoncarew) [inline packages/flutter_tools/test/web/devices_test.dart]: I agree w/ @DanTup's comment - `chrome` seems like a good ID to use here.

**Review by tvolkert — APPROVED** [review]
LGTM

File: packages/flutter_tools/test/web/devices_test.dart (hunk 2 above)
Comment (jonahwilliams) [reply packages/flutter_tools/test/web/devices_test.dart]: Renamed to chrome

**Comment by jonahwilliams** [comment]
Renamed the `web/web` device to `Chrome/chrome`

**Comment by devoncarew** [comment]
Thanks, this works great:

Screen Shot 2019-06-21 at 2 38 06 PM

**Comment by jonahwilliams** [comment]
Well, forgot to update a devicelab test, but I will reland soon

**Pull request merged**